src/containers/ListingPage/ListingPage.duck.js

```diff
@@ -190,5 +229,19 @@ export const loadData = (params, search) => dispatch => {
   if (params.variant === LISTING_PAGE_PENDING_APPROVAL_VARIANT) {
     return dispatch(showListing(listingId, true));
   }
-  return Promise.all([dispatch(showListing(listingId)), dispatch(fetchReviews(listingId))]);
+
+  // fetch time slots for 90 days starting today
+  // as the booking can only be done for 90 days
+  // in the future due to Stripe limitations
+  const start = moment().toDate();
+  const end = moment()
+    .add(config.dayCountAvailableForBooking, 'days')
+    .toDate();
+  const timeSlotsParams = { listingId, start, end };
+
+  return Promise.all([
+    dispatch(showListing(listingId)),
+    dispatch(fetchTimeSlots(timeSlotsParams)),
```

Comment:
As discussed IRL, I think this call should be included when flagged somehow in config.js